clowdapp.yaml

```diff
@@ -0,0 +1,116 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ros
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ros
+  spec:
+    envName: ${ENV_NAME}
+    dependencies:
+    - rbac
+    pods:
+      - name: backend
+        minReplicas: 1
+        image: ${IMAGE}:${IMAGE_TAG}
+        command: ["bash"]
+        args: ["-c", "python -m manage db upgrade && python -m ros.api.main"]
+        resources:
+          requests:
+            cpu: ${CPU_REQUEST}
+            memory: ${MEMORY_REQUEST}
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: ${MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /api/ros/v0/status
+            port: 8080
```

Comment:
One thing to note is that your app needs to run on port `8000` now and your metrics will need to be exposed on `9000`. 

Docs for that decision are here: https://clouddot.pages.redhat.com/docs/dev/getting-started/migration/faq.html#_why_does_my_app_have_to_run_on_port_8000 

So you'll just need to update: https://github.com/RedHatInsights/ros-backend/blob/e056b7ac9b12bc334f185ad420b7c9bda4610f57/ros/api/main.py#L11 to be `8000`.